api/mesh/v1alpha1/dataplane_insight.proto

```diff
@@ -124,6 +124,9 @@ message Version {
 
   // Version of Envoy
   EnvoyVersion envoy = 2;
+
+  // Versions of other dependencies, i.e. CoreDNS
+  map<string, string> otherDependencies = 3;
```

Comment:
I think I'd call it just "dependencies", but it's up to you

Reply:
I agree, so I changed it do `dependencies` as suggested